src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -2322,6 +2324,143 @@ private void Window_PreviewMouseLeftButtonUp(object sender, MouseButtonEventArgs
             }
         }
 
+        private void GetStartedMenuItem_Click(object sender, RoutedEventArgs e)
+        {
+            ShowGetStartedGuidedTour();
+        }
+
+        /// <summary>
+        /// This method probably will be modified or deleted in the future when the GuideManager and Guide class are created
+        /// For now will be used just for testing/demo purposes since the popups will be created probably in the Guide class.
+        /// </summary>
+        private void ShowGetStartedGuidedTour()
+        {
+
+            Step.TotalSteps = 6;
```

Comment:
Can we define this 6 as const somewhere?

Reply:
@QilongTang This method (private void ShowGetStartedGuidedTour()) was added just for testing purposes, so Aabishkar will be able to validate all the PopupWindow UI implemented (when the SurveyWindow UI is finished also will be added in this method), then I'm not sure if is worth to create a constant for the value 6.
Thanks

Reply:
I see, then dont worry about it